seedfarmer/cli_groups/_bootstrap_group.py

```diff
@@ -100,11 +101,21 @@ def bootstrap() -> None:
     help="A qualifier to append to toolchain role (alpha-numeric char max length of 6)",
     required=False,
 )
+@click.option(
+    "--policy-arn",
+    "-pa",
+    help="""ARN of existing Policy to attach to Target Role (Deploymenmt Role)
```

Comment:
can you make this more clear about it being valid in single account structure (--as-target)?

Reply:
yes